man/docker-daemon.8.md

```diff
@@ -513,6 +513,16 @@ By default docker will pick up the zfs filesystem where docker graph
 
 Example use: `docker daemon -s zfs --storage-opt zfs.fsname=zroot/docker`
 
+## Btrfs options
+
+#### btrfs.min_free_space
+
+Specifies the size to use when creating the subvolume which is used for
+containers. If user uses disk quota for btrfs, docker will check the
+minimum free space when a new container is created.
```

Comment:
@thaJeztah  oh, now *btrfs.min_free_space* is only to set the minimum size of the subvolume (per container). Then if you specify the *size* option when creating a container, the value of *size* must not be smaller than *min_free_space*.  Maybe the variable name is not correct.